Jenkinsfile

```diff
@@ -332,6 +348,35 @@ pipeline {
                 checkout scm
             }
         }
+        stage('check_for_changed_files') {
+          steps {
+            script {
+	      if (env.BRANCH_NAME == 'master') {
```

Comment:
Do we want to use a `when` here too? This is more optional, but it would be nice to see that they're skipped on purpose on master, instead of just having no output.

Reply:
You mean something like
```
         when {
            expression { env.BRANCH_NAME != 'master' }
         }
         steps {
            script {
	      def all_files = new HashSet<String>()
```
Note, I am not sure if my syntax is correct here.

Reply:
Yep, that is correct.

Reply:
Handled with commit 25b6ffb7832d599209b8673be4d57589c7d0582b